.gitignore

```
@@ -2,4 +2,6 @@
 .tox
 dist/
 wideq_state.json
-__pycache__
\ No newline at end of file
+__pycache__
+# Ignore Mac system files
+.DS_store
```

```suggestion
.DS_Store
```

I think it's capitalized.

---

let me see

---

yes my bad